spec/controllers/reductions_controller_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'spec_helper'
+require 'ostruct'
+
+describe ReductionsController, :type => :controller do
+  let(:reductions) {
+    [
+      Reduction.create!(workflow_id: 1234, subject_id: 5678, reducer_key: 'r', data: '1'),
+      Reduction.create!(workflow_id: 1234, subject_id: 5678, reducer_key: 's', data: '2'),
+      Reduction.create!(workflow_id: 1234, subject_id: 6789, reducer_key: 'r', data: '3')
+    ]
+  }
+
+  def tamper_auth
```

Comment:
This method seems similar to `fake_session(admin: true)`. Could that be used instead? And every single test case seems to call it, so might as well just do `before { fake_session(admin: true) }`.